sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/OperationType.java

```diff
@@ -50,7 +50,7 @@ public boolean isWriteOperation() {
                 this == ExecuteJavaScript ||
                 this == Replace ||
                 this == Upsert ||
-                this == Update ||
```

Comment:
ditto

Reply:
Operation type update was renamed to Patch in Backend, .net.